app/rfid.py

```diff
@@ -11,21 +17,17 @@ def __init__(self):
             parity=serial.PARITY_NONE,
             stopbits=serial.STOPBITS_ONE,
             bytesize=serial.EIGHTBITS,
-            timeout=0,
+            timeout=10
         )
-        # ser.write(b'SD2\r\n')
+        # ser.write(b'SD2\r\n') # Sets transponder default to FDX-B(animal tag)
 
     def read(self):
-        # ser.reset_input_buffer()
+        ser.reset_input_buffer()
         ser.reset_output_buffer()
-        ser.write(b'rat\r\n')
-        a = ser.read(34).decode('utf-8')
-        if '?1' in a:
+        ser.write(b'sd2\r\n')
```

Comment:
it's good to provide links to manuals/documentation in a comment when using codes like this. I guess this refers to http://www.priority1design.com.au/rfidrw-e-usb.pdf?

Reply:
Done